### PR TITLE
Add prettier

### LINKED ISSUE #529: Docs precommit hook isn't working for everyone
Needs some investigation

### PR DESCRIPTION
**Pull request by chawes13**
Adds `prettier`, formats all relevant files, and sets up pre-commit hook to run eslint and prettier on only the files that have been modified and staged for commit.

Also adds the missing `prepare` script to install husky, which resolves #529 

## Author Checklist

- ~Add unit test(s)~
- [x] Update version in package.json (see the [versioning guidelines](https://github.com/LaunchPadLab/opex-public/blob/main/gists/npm-package-guidelines.md#pull-requests-and-deployments))
- ~Update documentation (if necessary)~
- ~Add story to storybook (if necessary)~
- [x] Assign dev reviewer

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn run docs && git add docs.md
+yarn run lint-staged
+yarn run docs && git add docs.md
```
Comment (danparnella) [inline .husky/pre-commit:5]: Just a question around husky. Is there any difference between creating this file vs adding something like this to `package.json` (besides that also running `docs && git add docs.md`)?
```
"husky": {
  "hooks": {
    "pre-commit": "lint-staged"
  }
},
```

**Review by danparnella — APPROVED** [review]
Had one question and obviously didn't check every file, but I trust `prettier` 😅.

File: .husky/pre-commit (hunk 1 above)
Comment (chawes13) [reply .husky/pre-commit:5]: Here's the release notes: https://github.com/typicode/husky/releases/tag/v6.0.0

Comment (danparnella) [reply .husky/pre-commit:5]: Awesome, thanks!

**Pull request merged**